k8s/images/nginx/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM byrnedo/alpine-curl
 
 # download all extra deps we need for the production container
-
 # templating executable
-RUN curl -L "https://github.com/gliderlabs/sigil/releases/download/v0.4.0/sigil_0.4.0_$(uname -sm|tr \  _).tgz" \ | tar -zxC /usr/bin
+COPY /k8s/images/nginx/download_sigil.sh /tmp/download_sigil.sh
+RUN chmod +x /tmp/download_sigil.sh
+RUN /tmp/download_sigil.sh
```

Comment:
Oof

Reply:
I moved it to a script, since complex env vas substitution doesn't work in a Dockerfile.